pandas/types/dtypes.py

```diff
@@ -98,25 +99,73 @@ class CategoricalDtypeType(type):
 class CategoricalDtype(ExtensionDtype):
 
     """
+    Type for categorical data with the categories and orderedness,
+    but not the values
+
+    .. versionadded:: 0.20.0
```

Comment:
this has been around for quite some time, you are adding parameter support in 0.20.0